src/main/java/TaskManager.java

```diff
@@ -0,0 +1,289 @@
+import java.io.*;
+import java.time.LocalDate;
+import java.util.Locale;
+import java.util.Objects;
+
+public class TaskManager {
+    Task[] taskList = new Task[100];
+    int i = 0;
+    File txtFile;
+    String path;
+
+    TaskManager() {
+        File txtFile;
+        System.out.println(System.getProperty("user.dir"));
+        String path = System.getProperty("user.dir") + "\\src\\main\\data\\duke.txt";
+        txtFile = new File(path);
+        this.txtFile = txtFile;
+        this.path = path;
+
+        if (!txtFile.exists()) {
+            //create new file
+            try {
+                txtFile.createNewFile();
+                this.txtFile = txtFile;
+            } catch (IOException e) {
+                System.out.println("An error occurred.");
+                e.printStackTrace();
+            }
+        } else {
+            try {
+                BufferedReader br = new BufferedReader(new FileReader(txtFile));
+                String txtLine;
+                txtLine = br.readLine();
+                while(txtLine != null) {
+                    String[] segments = txtLine.split("\\|");
+                    executeCommand(segments[2]);
+                    if (Objects.equals(segments[0], "1")) {
+                        String input = "done " + segments[1];
+                        executeCommand(input);
+                    }
+                    txtLine = br.readLine();
+                }
+            } catch (IOException e) {
+                System.out.println("ERROR OCCURRED");
+            }
+        }
+    }
+
+    public void executeCommand(String inData) {
+        int inDataLength = inData.length();
+        if (!Objects.equals(inData.toLowerCase(), "bye")) {
+            if (Objects.equals(inData, "list")) {
+                listAll();
+            } else if (inData.contains("todo")) {
```

Comment:
Nice! I like the way `inData.contains("todo")` reads as "In data contains todo" in english! Very comfortable to read 👍